tests/execution/test-http-batchKey-rename-nested-simple.md

```diff
@@ -0,0 +1,98 @@
+# Http with args as body
+
+```graphql @config
+schema
+  @server(port: 8000)
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", batch: {maxSize: 1000, delay: 10}) {
+  query: Query
+}
+
+type Query {
+  transactions: TransactionsResponse @http(path: "/transactions")
+}
+
+type TransactionsResponse {
+  transactions: [Transaction!]!
```

Comment:
This is the actual user scenario. However it's a very complicated one. Can we create a simpler tests, using Foo Bar perhaps?